src/incubator/f5-bigip-ctlr/README.md

```diff
@@ -1,11 +1,25 @@
-# Alpha status
+# Helm Chart for the F5 BIG-IP Controller
 
-This chart is in an alpha status. It can be used to deploy the __development__ image of the [F5 k8s-bigip-ctlr](https://github.com/F5Networks/k8s-bigip-ctlr). The stable image will be available in a stable chart.
+This chart simplifies repeatable, versioned deployment of the [F5 BIG-IP Controller for Kubernetes](http://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/latest/).
 
-To deploy the k8s-bigip-ctlr with this chart the following must be true of your cluster:
-- A secret for the BigIP must exist and be named as in the values file, or passed using `--set` or `-f your_values.yaml`.
-- The partition must exist and be named as in the values file, or passed using `--set` or `-f your_values.yaml`.
-- The BigIP management url must be passed as an argument using `--set` or configured in a values file.
+## Requirements
+
+The chart presumes:
+- You have added a BIG-IP device to either your [Kubernetes](http://clouddocs.f5.com/containers/latest/kubernetes/kctlr-use-bigip-k8s.html) or [OpenShift](http://clouddocs.f5.com/containers/v2/openshift/kctlr-use-bigip-openshift.html) cluster.
+- A partition (other than common) exists on your BIG-IP for the instance of the controller to manage.
+- A secret exists in your cluster to access the BIG-IP.
+
+The default values for the partition and the secret to access the BIG-IP are:
+- `f5-bigip-ctlr` and 
+- `f5-bigip-ctlr-login` 
+
+These values can be easily updated using either `--set <param>=<value>` or `-f <values-file.yaml>`. See [customizing the chart before installing](https://docs.helm.sh/using_helm/#customizing-the-chart-before-installing) for more details.
```

Comment:
Be sure to change these if they differ from your actual partition and Secret names, using `--set <param>=<value>` or `-f <values-file.yaml>` as appropriate.